docs/user_workflow.adoc

```diff
@@ -0,0 +1,423 @@
+-c 'UUID' -c 'State' -c 'Action'
++--------------------------------------+-----------+---------------------------+
+| UUID                                 | State     | Action                    |
++--------------------------------------+-----------+---------------------------+
+| 01774d02-00a9-4f34-a6f7-6b2264b8970c | SUCCEEDED | change_nova_service_state |
+| 8573aa4e-6fac-4002-8de6-569e72d4dca3 | SUCCEEDED | migrate                   |
+| 6d88ea4a-012b-4cb8-af86-8e699c6c2738 | SUCCEEDED | migrate                   |
+| fa2827c6-78f8-48b8-8f8a-734b9f170841 | SUCCEEDED | migrate                   |
+| 4009c44d-9af6-4a6e-91dd-96bd4f17abd5 | SUCCEEDED | migrate                   |
+| e3dc2dec-74fc-4f16-b76d-c4b99acb1b01 | SUCCEEDED | change_nova_service_state |
++--------------------------------------+-----------+---------------------------
+----
++
+You can check that the instances have been actually consolidated in one of your hosts by
+listing the instances (VMs) on each one of your hypervisors.
++
+[,console]
+----
+# List the hypervisors:
+$ openstack hypervisor list -c 'Hypervisor Hostname' -c 'State'
++-------------------------------+-------+
+| Hypervisor Hostname           | State |
++-------------------------------+-------+
+| compute2.ctlplane.localdomain | up    |
+| compute1.ctlplane.localdomain | up    |
++-------------------------------+-------+
++
+# Note that the output below lists all instances on one host:
+$ openstack server list --long -c Name -c 'Host' --project demo
++--------+-------------------------------+
+| Name   | Host                          |
++--------+-------------------------------+
+| test_7 | compute2.ctlplane.localdomain |
+| test_6 | compute2.ctlplane.localdomain |
+| test_5 | compute2.ctlplane.localdomain |
+| test_4 | compute2.ctlplane.localdomain |
+| test_3 | compute2.ctlplane.localdomain |
+| test_2 | compute2.ctlplane.localdomain |
+| test_1 | compute2.ctlplane.localdomain |
+| test_0 | compute2.ctlplane.localdomain |
++--------+-------------------------------+
+----
+
+== Watcher Workflow (Horizon Dashboard UI)
+
+=== Optimization Scenario II: Workload balancing/stabilization
+
+This workflow is demonstrated through actions in the `Horizon UI`.
+Example screenshots are added where necessary to further explain the user flow.
+
+.Procedure
+. Login in the `Horizon` dashboard with credentials which would enable the `Admin`
+role in one or more projects.
+. Make sure you have the `Administration` menu enabled by selecting the project
+where you have the `Admin` role assigned in the Projects tab:
++
+
+image::admin_role.png[]
+
++
+. In the Admin menu a new panel `Optimization` should be available:
++
+
+image::optimize_menu.png[]
+
++
+. In the `Audit Templates` panel, click on `Create Template` button. This will open a
+`Create Audit Template` window. Add a new `Audit Template` called *AuditTemplateWorkloadStabilization*
+with the goal *Workload Balancing* and the Strategy *Workload stabilization*.
```

Comment:
We could add the link to the strategy doc also here: https://docs.openstack.org/watcher/2024.2/strategies/workload-stabilization.html